StructureDefinition/England-PractitionerRole.xml

```diff
@@ -0,0 +1,353 @@
+      <mustSupport value="true" />
+    </element>
+    <element id="PractitionerRole.identifier:sdsRoleProfileID">
+      <path value="PractitionerRole.identifier" />
+      <sliceName value="sdsRoleProfileID" />
+      <max value="1" />
+      <mustSupport value="true" />
+    </element>
+    <element id="PractitionerRole.identifier:sdsRoleProfileID.system">
+      <path value="PractitionerRole.identifier.system" />
+      <fixedUri value="https://fhir.nhs.uk/Id/sds-role-profile-id" />
+    </element>
+    <element id="PractitionerRole.identifier:nhsbsaSpuriousCode">
+      <path value="PractitionerRole.identifier" />
+      <sliceName value="nhsbsaSpuriousCode" />
+      <max value="1" />
+      <mustSupport value="true" />
+    </element>
+    <element id="PractitionerRole.identifier:nhsbsaSpuriousCode.system">
+      <path value="PractitionerRole.identifier.system" />
+      <fixedUri value="https://fhir.hl7.org.uk/Id/nhsbsa-spurious-code" />
+    </element>
+    <element id="PractitionerRole.practitioner">
+      <path value="PractitionerRole.practitioner" />
+      <definition value="This **MUST** either be a reference to a Practitioner resource or an identifier reference plus a display. In both cases the referenced resource or identfier reference should contain a professional code. &#xD;&#xA;&#xD;&#xA;It is recommended an *identifier reference* and *display* (practioner full name) is included. This identifier should be the Practitioner's primary professional code (see `NHSDigital-Practitioner` for details on professional codes). &#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&quot;practitioner&quot;: {&#xD;&#xA;        &quot;display&quot;: &quot;Dr Smith&quot;,&#xD;&#xA;        &quot;identifier&quot;: {&#xD;&#xA;            &quot;system&quot;: &quot;https://fhir.hl7.org.uk/Id/gmc-number&quot;,&#xD;&#xA;            &quot;value&quot;: &quot;C9876543&quot;&#xD;&#xA;          }&#xD;&#xA;    },&#xD;&#xA;```&#xD;&#xA;&#xD;&#xA;For EPS, currently a *Resource reference* is required.&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&quot;practitioner&quot;: {&#xD;&#xA;        &quot;reference&quot;: &quot;urn:uuid:1557E58E-3B1E-41DD-B3B5-D4D393DC5A3D&quot;,&#xD;&#xA;        &quot;display&quot;: &quot;Dr Smith&quot;&#xD;&#xA;    },&#xD;&#xA;```" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Practitioner" />
```

Comment:
As per discussion, should not be adding references

Reply:
Pushed change